apps-rendering/src/components/liveblogHeader.tsx

```diff
@@ -26,10 +29,7 @@ const LiveblogHeader: FC<Props> = ({ item }) => {
 
 	return (
 		<header>
-			<Container
-				element="div"
-				backgroundColor={background.headline(format)}
-			>
+			<Container element="div" css={headlineBackgroundColour(format)}>
```

Comment:
I was thinking we'd have to implement a `backgroundColorDark` prop in the API of this container, but actually this works and it's very simple 🤔

I'm happy to go with this approach, but FYI @SiAdcock I think you'd be interested to know this is how the API is being used to achieve dark mode support?